src/App/index.tsx

```diff
@@ -153,32 +148,12 @@ function App() {
         },
         {
           path: RoutePaths.ProjectCreation,
-          children:
-            parseFlag("VITE_CAN_CREATE_PROJECT") && isLoggedIn
-              ? [
-                  {
-                    index: true,
-                    element: <ProjectIntroPage />,
-                  },
-                  {
-                    path: "organizations",
-                    element: <GithubOrganizationPage />,
-                  },
-                  {
-                    path: "repository",
-                    element: <GithubRepositoryPage />,
-                  },
-                  {
-                    path: "informations",
-                    element: <ProjectInformationsPage />,
-                  },
-                ]
-              : [
-                  {
-                    index: true,
-                    element: <Navigate to={RoutePaths.Projects} />,
-                  },
-                ],
+          element:
+            parseFlag("VITE_CAN_CREATE_PROJECT") && isLoggedIn ? (
+              <ProjectCreation />
+            ) : (
+              <Navigate to={RoutePaths.Projects} />
+            ),
```

Comment:
Maybe handle `isLoggeIn` with `ProtectedRoute` component directly ? And extends the component to accept a `featureFlag` to also pass `VITE_CAN_CREATE_PROJECT ` ? 

```suggestion
 element: 
            <ProtectedRoute requiredRole={HasuraUserRole.RegisteredUser} featureFlag="VITE_CAN_CREATE_PROJECT">
                <ProjectCreation />
            </ProtectedRoute>
```